src/lib/config.h

```diff
@@ -18,11 +18,30 @@
 
 #include "build_config.h"
 
+#ifdef _WIN32
+#    ifdef BUILD_NEURAL_POINTS_DLL
+#        define _NP_API __declspec(dllexport)
+#        define _NP_INLINE_API inline
+#        define _NP_EXT_API
+#    else
+#        define _NP_API __declspec(dllimport)
+#        define _NP_INLINE_API
+#        define _NP_EXT_API extern
+#    endif
+
+#else
+// UNIX, ignore _API
+#    define _NP_API
+#    define _NP_EXT_API
+#    define _NP_INLINE_API inline
+#endif
+
+
 namespace Saiga
 {
 class Camera;
 }
-inline Saiga::Camera* camera;
+_NP_INLINE_API _NP_API Saiga::Camera* camera;
```

Comment:
can we just make the global variable not inline?

Reply:
Didn't work for me. Windows was a bit tricky with this part